### DISCUSSION TITLE
Sonoff Sensor stops reporting after adding Philips Hue Globe

### DISCUSSION THREAD

**Discussion by liveitnerd** [Koenkk/zigbee2mqtt#11894] [Devices]
Hey all,

I've been using Zigbee2MQTT successfully for a few months now installed as an addon to Home Assistant.  The Zigbee Coordinator is a Sonoff Zigbee 3.0 USB Plus device and has performed well and can talk to all of the sensors etc in the house.

I decided to try and migrate the lights from the Philips Hue Bridge to Zigbee2MQTT so I could remove the duplication.  After adding hte first globe one of the temperature sensors stopped reporting most of it's parameters.  It says it's still online but there's no update on LQI, Temperature, or Humidity.  Only after removing the hue globe and the temperature sensor from Zigbee2MQTT and resetting the coordinator could I get the sensor to join the zigbee network and start reporting all of it's parameters.  

At the moment the Zigbee network is the Sonoff Zigbee 3.0 coordinator, 2x Temp Sensors (SNZB-02), 2x Reed Switches (SNZB-04), and 4x push buttons (SNZB-01).  I am hoping to migrate the 4x Philips Hue bulbs over but there's not much point if they are going to kill off some of the sensors.

Any suggestion as to what I might be doing wrong or what I can check would be greatly appreciated.

A huge thanks in advance.

**Comment by premyslc** [comment]
I have same problem with TS0505B (TechToy Smartbulb). May be some problem with bulbs and broken routing. I will try it without bulb.

**Comment by premyslc** [comment]
No issue since i removed smart bulb from the network. Seems to be some problem with bulbs. I will try to return it to the seller. ;-)
